### DISCUSSION TITLE
Pre-gate switches with Easy BRD

### DISCUSSION THREAD

**Discussion by genghisosmosis** [Enraged-Rabbit-Community/ERCF_v2#158] [Q&A]
Anyone any experience of using the pre-gate switches with the "spare" single connector on the Easy-BRD? IF the switches are all wired NC by default, is it safe to assume that a "trigger" can be detected just on this one pin? Or am I raving ?

**Comment by norris108** [comment]
You'll need a connector per pre-gate switch; otherwise, Happy Hare won't know _which_ gate has triggered

**Comment by genghisosmosis** [comment]
My thought process was that it's really ever going to trigger on the
currently "active" channel anyway ( apart from some edge cases maybe ) , or
am I missing something? I realise this might need a bit of a tweak
software side somewhere..
